boilerplates/be-docker-plain/Jenkinsfile

```diff
@@ -0,0 +1,55 @@
+def final projectId = '@project_id@'
+def final componentId = '@component_id@'
+def final credentialsId = "${projectId}-cd-cd-user-with-password"
+def sharedLibraryRepository
+def dockerRegistry
+node {
+  sharedLibraryRepository = env.SHARED_LIBRARY_REPOSITORY
+  dockerRegistry = env.DOCKER_REGISTRY
+}
+
+library identifier: 'ods-library@0.1-latest', retriever: modernSCM(
```

Comment:
This should point to `production` on the `master` branch, not to `0.1-latest`.

Reply:
The quickstarter will first go to pro on prem - then i'll upgrade ok?

Reply:
hi! @clemensutschig : the library is defined to always point to production by default. On prem too. For testing or for specific library identifiers users can later define it.